docs/tutorials/AddRemark.md

```diff
@@ -295,7 +295,8 @@ While the changes to code may be minimal, the test data will have to be updated
 
 <div markdown="span" class="alert alert-warning">
 
-:exclamation: You must delete AddressBook’s storage file located at `/data/addressbook.json` before running it! Not doing so will cause AddressBook to default to an empty address book!
+:exclamation: You must delete ReadyBakey’s storage file located at `/data/readybakey.json` before running it! Not 
+doing so will cause AddressBook to default to an empty address book!
```

Comment:
Should this be changed to ReadyBakey as well?

Reply:
Updated this in my latest commit. Thanks!